kentikapi/client.go

```diff
@@ -78,26 +75,67 @@ type Config struct {
 	// A Timeout of zero means no timeout.
 	// Default: 100 seconds.
 	Timeout *time.Duration
-	// DisableTLS disables TLS for client connections.
-	// It has effect only on gRPC services: CloudExportAdmin, SyntheticsAdmin and SyntheticsData.
-	DisableTLS bool
 }
 
 type RetryConfig = httputil.RetryConfig
 
+type ConfigOption func(*Config)
+
+func WithAPIURL(apiURL string) ConfigOption {
+	return func(c *Config) {
+		c.APIURL = apiURL
+	}
+}
+
+func WithAuthEmail(authEmail string) ConfigOption {
+	return func(c *Config) {
+		c.AuthEmail = authEmail
+	}
+}
+
+func WithAuthToken(authToken string) ConfigOption {
+	return func(c *Config) {
+		c.AuthToken = authToken
+	}
+}
+
+func WithTimeout(timeout *time.Duration) ConfigOption {
```

Comment:
```suggestion
func WithTimeout(timeout time.Duration) ConfigOption {
```
It doesn't make sense to take a pointer here

Reply:
Done